Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM debian:wheezy
+FROM gliderlabs/alpine:3.3
 MAINTAINER Matt Bentley <mbentley@mbentley.net>
 
-RUN (apt-get update && apt-get install -y virt-manager ssh ssh-askpass)
+RUN (apk --update add virt-manager openssh-client && apk --update add openssh-askpass --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted)
```

Comment:
Looking at https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache indicates that there might a smother variant. :)